express-api/src/routes/propertiesRouter.ts

```diff
@@ -27,6 +29,10 @@ router.route('/search/geo').get(activeUserCheck, catchErrors(getPropertiesForMap
 router.route('/search/page').get(activeUserCheck, catchErrors(getPropertiesPaged));
 router.route('/search/page/filter').post(activeUserCheck, catchErrors(getPropertiesPagedFilter));
 
+const upload = multer({ dest: 'uploads/' });
```

Comment:
Does this mean it will keep a local copy of the file in the container? Or it's just temporary?

Reply:
It does temporarily create a copy of the file in the container but propertiesController.ts:210 will attempt to destroy it before exiting after processing the file.